### ISSUE TITLE
Should freezing still leave the features togglable?

### ISSUE THREAD

**Issue by madig** [twardoch/fonttools-utils#19]
If I freeze `ss01` in a font, should toggling it do the reverse?

I think this was once the case but I might be misremembering...

**Comment by twardoch** [comment]
I think it would make sense. This would of course require reversing the sub rules. In my initial implementation I only remapped the Unicodes, so the original encoded glyphs are orphaned.

But if you do modify GSUB, make it optional. I use the freezer to build TTCs: Basically, alternative cmap & name tables are added as extra "font API" tables but the rest remains the same. This makes the TTC only negligibly larger than the original font. But with changed GSUBs, this would increase the size a lot.

**Comment by madig** [comment]
Mh. Wouldn't it be enough to swap the `sub ... by ...` lookups inside the feature? 🤔

**Comment by twardoch** [comment]
Yeah, that's what I mean. I've never done this and if you feel like doing it, please kindly make this optional (onable or offable, I don't care) for the TTC reasons ☺️

**Comment by madig** [comment]
Just wondering, would it increase TTC size because you might not want to apply it to all contained fonts?
